xapian-core/tests/api_cluster.cc

```diff
@@ -80,3 +104,27 @@ DEFINE_TESTCASE(round_robin1, backend)
     }
     return true;
 }
+
+DEFINE_TESTCASE(stem_stopper1, backend)
+{
+    Xapian::Stem stemmer("english");
+    Xapian::StemStopper stopper(stemmer);
+    std::string term = "the";
+    stopper.add(term);
+    TEST(stopper(term));
+    TEST(stopper('Z' + stemmer(term)));
+
+    term = "because";
+    stopper.set_stemming_strategy(Xapian::StemStopper::STEM_ALL_Z);
+    stopper.add("because");
+    TEST(!stopper(term));
+    TEST(stopper('Z' + stemmer(term)));
+
+    term = "and";
+    stopper.set_stemming_strategy(Xapian::StemStopper::STEM_NONE);
+    stopper.add(term);
+    TEST(stopper(term));
+    TEST(!stopper('Z' + stemmer(term)));
+
+    return true;
+}
```

Comment:
There's no newline character at the end of this file (the red slashed circle and arrow thing in the git hub diff view shows this).